ui/webui/test/helpers/installer.py

```diff
@@ -63,3 +64,22 @@ def wait_current_page(self, page):
             self.browser.wait_visible(".pf-c-progress-stepper")
         else:
             self.browser.wait_visible(f"#{page}.pf-m-current")
+
+    def check_prerelease_info(self, is_expected=None):
+        """ Checks whether the pre-release information is visible or not.
+
+        If is_expected is not set, the expected state is deduced from .buildstamp file.
+
+        :param is_expected: Is it expected that the info is visible or not, defaults to None
+        :type is_expected: bool, optional
+        """
+        if is_expected is not None:
+            value = str(is_expected)
+        else:
+            value = self.machine.execute("grep IsFinal= /.buildstamp").split("=", 1)[1]
```

Comment:
I don't see the value of introducing an additional parameter here, when it isn't used at all. Otherwise LGTM.

Reply:
Well, it isn't used yet, we will use it in integration tests.